lib/src/http/http.dart

```diff
@@ -24,35 +25,55 @@ class Http {
   /// Secure cookies (credentials) will be included (in the browser) if
   /// [withCredentials] is true.
   static Future<Response> delete(Uri uri,
-          {Map<String, String> headers, bool withCredentials}) =>
-      _createRequest(uri, headers: headers, withCredentials: withCredentials)
+          {Map<String, String> headers,
+          TransportPlatform transportPlatform,
```

Comment:
I had forgotten to add support for specifying the transport platform when using the static `Http` methods